sklearn_genetic/genetic_search.py

```diff
@@ -342,7 +342,7 @@ def _register(self):
         self._pop = self.toolbox.population(n=self.population_size)
         self._hof = tools.HallOfFame(self.keep_top_k)
 
-        self._stats = tools.Statistics(lambda ind: ind.fitness.values)
+        self._stats = tools.Statistics(getter_fittness_values)
```

Comment:
@imxtx here the name also should be ind_fitness_values

Reply:
I'm so stupid...

Reply:
don't worry, those things happens